### ISSUE TITLE
Custom component partial gets replaced by built-in partial after AJAX request

### ISSUE THREAD

**Issue by guus-frenken** [OFFLINE-GmbH/oc-mall-plugin#832]
When I create a custom partial for certain Mall components, it works initially but after executing an AJAX request that calls renderPartial, the custom partial gets replaced with the built-in component from the plugin.

Example: On the myaccount page, I added an alias to my custom partial of the 'addressList' component. When I change my default shipping / billing address, it replaces my custom partial with the built-in one.

When I check out the [code](https://github.com/OFFLINE-GmbH/oc-mall-plugin/blob/95a916f2ec58fb720b0fe561f897a2541e3b9f6f/components/AddressList.php#L95), I see that the `renderPartial` method uses `$this->alias`. Shouldn't this be addressing my custom partial?

**Comment by tobias-kuendig** [comment]
> Shouldn't this be addressing my custom partial?

Yes, exactly. That's why we use the alias in the first place. What are your software versions?

**Comment by guus-frenken** [comment]
I am using Mall v1.14.3 on October CMS v2.1.5

My custom partial is loaded in the myaccount page like this: `[addressList qtAddressList]`.
I also have custom partials for the 'myAccount' component (as well as every other component), I load the custom 'addressList' partial in 'qtMyAccount/pages/addresses.htm' like this: `{% component 'qtAddressList' %}`

**Comment by tobias-kuendig** [comment]
I'm closing this issue since it has been stale for more than 4 years and a lot of the code has changed in the meantime. If the problem still exists in the current version, please reply.
